x/evm/types/params.go

```diff
@@ -40,28 +41,36 @@ func ParamKeyTable() paramtypes.KeyTable {
 }
 
 // NewParams creates a new Params instance
-func NewParams(evmDenom string, enableCreate, enableCall bool, config ChainConfig, extraEIPs ...int64) Params {
+func NewParams(evmDenom string, enableCreate, enableCall bool, config ChainConfig, minGasMultiplier sdk.Dec, extraEIPs ...int64) Params {
 	return Params{
-		EvmDenom:     evmDenom,
-		EnableCreate: enableCreate,
-		EnableCall:   enableCall,
-		ExtraEIPs:    extraEIPs,
-		ChainConfig:  config,
+		EvmDenom:         evmDenom,
+		EnableCreate:     enableCreate,
+		EnableCall:       enableCall,
+		ExtraEIPs:        extraEIPs,
+		ChainConfig:      config,
+		MinGasMultiplier: minGasMultiplier,
 	}
 }
 
 // DefaultParams returns default evm parameters
 // ExtraEIPs is empty to prevent overriding the latest hard fork instruction set
 func DefaultParams() Params {
 	return Params{
-		EvmDenom:     DefaultEVMDenom,
-		EnableCreate: true,
-		EnableCall:   true,
-		ChainConfig:  DefaultChainConfig(),
-		ExtraEIPs:    nil,
+		EvmDenom:         DefaultEVMDenom,
+		EnableCreate:     true,
+		EnableCall:       true,
+		ChainConfig:      DefaultChainConfig(),
+		ExtraEIPs:        nil,
+		MinGasMultiplier: DefaultMinGasMultiplier(),
 	}
 }
 
+// DefaultMinGasMultiplier returns default value for MinGasMultiplier.
+// Default value is 0.5 or 50%
+func DefaultMinGasMultiplier() sdk.Dec {
```

Comment:
Can we make this a variable instead? Otherwise you can't change the default value when executing the upgrade